app/models/company.rb

```diff
@@ -31,7 +31,9 @@ class Company < ApplicationRecord
   has_many :payments, dependent: :destroy
   accepts_nested_attributes_for :payments
 
-  has_many :business_categories, through: :shf_applications
+  has_many :business_categories,
```

Comment:
The Business Categories that should be shown is based on the _membership_ status, not the application status.

Reply:
The Admin needs to be able to see those I belive.  (The Admin always needs to be able to see everything.)  
Use case:  a member whose membership has expired yesterday.  I can imagine that an admin needs to be able to see that a company no longer has a particular category that (expired) member contributed.  
Perhaps the admin sees the category in a different way or section (IOW, the category is distinguished in some way so the admin realizes it is no longer active/current.)

I would be fine making a note of this _possibility_ in the comments of the code and fetching the business categories only for a current member.

Reply:
I'm not sure I understand - if a category is associated with a _rejected_ application, right now we would not know if that application was _never accepted_ - and thus the category was never associated with the company - or was rejected somehow after having been accepted.  What are you thinking about?

Reply:
The specific use case I'm thinking about is a member (which means they have met _all_ requirements for membership) that is paid up to and including July. 10th.  They are the only one with the 'groomer' category for a company with many people.
On July. 11 they are no longer a member.   The admin should see that the "groomer" category exists for that person, but because that person is _recently_ not a member, that category is not going to show to the public.
We've had a number of situations "recently" being a member is an important state, for the  'recent' member as well as the admin.